integration-tests/src/chains/asset_hub_paseo/mod.rs

```diff
@@ -13,27 +15,28 @@ decl_test_parachains! {
 	pub struct AssetHubPaseo {
 		genesis = genesis::genesis(),
 		on_init = {
-			asset_hub_paseo_runtime::AuraExt::on_initialize(1);
+			asset_hub_runtime::AuraExt::on_initialize(1);
 		},
-		runtime = asset_hub_paseo_runtime,
+		// Note: using polkadot as stopgap until paseo updated to polkadot sdk v1.14.0
+		runtime = asset_hub_runtime,
 		core = {
-			XcmpMessageHandler: asset_hub_paseo_runtime::XcmpQueue,
-			LocationToAccountId: asset_hub_paseo_runtime::xcm_config::LocationToAccountId,
-			ParachainInfo: asset_hub_paseo_runtime::ParachainInfo,
+			XcmpMessageHandler: asset_hub_runtime::XcmpQueue,
+			LocationToAccountId: asset_hub_runtime::xcm_config::LocationToAccountId,
+			ParachainInfo: asset_hub_runtime::ParachainInfo,
 			MessageOrigin: cumulus_primitives_core::AggregateMessageOrigin,
 		},
 		pallets = {
-			PolkadotXcm: asset_hub_paseo_runtime::PolkadotXcm,
-			Assets: asset_hub_paseo_runtime::Assets,
-			ForeignAssets: asset_hub_paseo_runtime::ForeignAssets,
-			Balances: asset_hub_paseo_runtime::Balances,
+			PolkadotXcm: asset_hub_runtime::PolkadotXcm,
+			Assets: asset_hub_runtime::Assets,
+			ForeignAssets: asset_hub_runtime::ForeignAssets,
+			Balances: asset_hub_runtime::Balances,
 		}
 	},
 }
 
 // AssetHubPaseo implementation
 impl_accounts_helpers_for_parachain!(AssetHubPaseo);
 impl_assert_events_helpers_for_parachain!(AssetHubPaseo);
-impl_assets_helpers_for_parachain!(AssetHubPaseo, Paseo);
-impl_foreign_assets_helpers_for_parachain!(AssetHubPaseo, Paseo);
+impl_assets_helpers_for_parachain!(AssetHubPaseo);
+impl_foreign_assets_helpers_for_parachain!(AssetHubPaseo, xcm::v3::Location);
 impl_xcm_helpers_for_parachain!(AssetHubPaseo);
```

Comment:
Should this not be `Assethub`?

Why on line 40 & 41 not `Polkadot`?

Reply:
- Line 40: Since this commit: https://github.com/paritytech/polkadot-sdk/pull/3543/files, the `impl_assets_helpers_for_parachain` no longer asks for the Relaychain in the macro.
- Line 41: I follow Polkadot runtime that does not use Polkadot: https://github.com/polkadot-fellows/runtimes/blob/main/integration-tests/emulated/chains/parachains/testing/penpal/src/lib.rs#L64

Reply:
Technically speaking the specification can be important in the context of bridges, whereby you could have Asset Hub on Polkadot and Kusama.

As we arent there yet, I would be fine with just `AssetHub`, but could perhaps be out of scope of the PR which is just fixing tests. Up to Tin on whether to address here.

Line 41 is due to the API changing with xcm v4, and AH using xcm v3. 40 is also an API change in the emulated tests macros.